doc/whats_new/v1.2.rst

```diff
@@ -2,6 +2,31 @@
 
 .. currentmodule:: sklearn
 
+.. _changes_1_2_1:
+
+Version 1.2.1
+=============
+
+**In Development**
+
+Changed models
+--------------
+
+The following estimators and functions, when fit with the same data and
+parameters, may produce different models from the previous version. This often
+occurs due to changes in the modelling logic (bug fixes or enhancements), or in
+random sampling procedures.
+
+Changelog
+---------
+
+:mod:`sklearn.utils`
+...................
```

Comment:
The [ci is failing here](https://github.com/scikit-learn/scikit-learn/actions/runs/3657945865/jobs/6182113225):

```suggestion
:mod:`sklearn.utils`
....................
```